templates/base.html

```diff
@@ -79,12 +79,11 @@
         <script src="{{ static('libs/jquery-cookie.js') }}"></script>
         <script src="{{ static('libs/jquery-taphold.js') }}"></script>
         <script src="{{ static('libs/jquery.unveil.js') }}"></script>
-        <script src="{{ static('libs/moment.js') }}"></script>
```

Comment:
Since we no longer need `moment.js`, we should probably delete https://github.com/DMOJ/site-assets/blob/master/moment.js